save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPluginConfig.kt

```diff
@@ -147,7 +151,8 @@ data class WarnPluginConfig(
             newMessageCaptureGroupOut,
             exactWarningsMatch ?: true,
             testName,
-            newDefaultLineMode
+            newDefaultLineMode,
+            newLinePlaceholder,
```

Comment:
I suggest not to create extra variables, when we have single line expressions

```suggestion
            defaultLineMode ?: false,
            linePlaceholder ?: "line",
```

Same сan be said for `newBatchSize`, `newBatchSeparator`